src/pydap/tests/data/dmrs/SimpleGroup.dmr

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Sample xml document. pydap.responses.dmr is able to generate this file from an array created interactively. I added the comparison (assertEqual) as a test